src/SFA.DAS.Assessor.Functions/Domain/Print/Services/PrintingJsonCreator.cs

```diff
@@ -123,7 +128,13 @@ public void Create(int batchNumber, List<CertificateResponse> certificates, stri
             using (var mystream = new MemoryStream(array))
             {
                 _logger.Log(LogLevel.Information, "Sending Certificates to print Json ....");
-                _fileTransferClient.Send(mystream, fileName);
+                _fileTransferClient.Send(mystream, file);
+            }
+
+            // update certificates status
+            foreach(var certificate in certificates)
+            {
+                certificate.Status = "Sent to Printer";
```

Comment:
What is "Sent to Printer' used for - it's a bit like 'SentToPrinter' the new certificate status in Assessor - enough to be confusing if it's not for that - I can't see how it's used though

Reply:
It's the status of the certificate at that point.  So a certificate has one of the following states.
"Sent to printer"
"Printed"
"Delivered"
"NotDelivered"

Do you think it should be "SentToPrinter" rather than "Sent to Printer" ?